packages/flutter/lib/src/material/badge.dart

```diff
@@ -184,8 +186,8 @@ class Badge extends StatelessWidget {
     final Widget badge;
     final bool hasLabel = label != null;
     if (hasLabel) {
-      final double minSize =
-          effectiveWidthOffset = largeSize ?? badgeTheme.largeSize ?? defaults.largeSize!;
+      final double minSize = effectiveWidthOffset =
```

Comment:
Seems this change and below are just formatting. We might need to revert these changes to fix the failed `linux_analyer` test:)